src/Tasks/Microsoft.NET.Build.Tasks/ProcessFrameworkReferences.cs

```diff
@@ -634,6 +635,15 @@ private string GetPackPath(string packName, string packVersion)
         {
             IEnumerable<string> GetPackFolders()
             {
+                if (!string.IsNullOrEmpty(NetCoreRoot) && !string.IsNullOrEmpty(NETCoreSdkVersion))
+                {
+                    if (WorkloadInstall.IsUserLocal(NetCoreRoot, NETCoreSdkVersion))
+                    {
+                        // TODO: should DotnetUserProfileFolderPath come from a property?
+                        yield return Path.Combine(CliFolderPathCalculator.DotnetUserProfileFolderPath, "packs");
```

Comment:
Probably there is no need to add a property for this, and `DotnetUserProfileFolderPath` can be used directly here?